src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaLocationManager.java

```diff
@@ -91,13 +97,23 @@ private void init() {
     @Scheduled(fixedDelay = 30000)
```

Comment:
I don't know. That's what is was, I haven't touched it but was asking myself the same.